src/client_side.cc

```diff
@@ -4103,13 +4116,25 @@ ConnStateData::checkLogging()
 bool
 ConnStateData::mayTunnelUnsupportedProto()
 {
-    return Config.accessList.on_unsupported_protocol
-#if USE_OPENSSL
-           &&
-           ((port->flags.isIntercepted() && port->flags.tunnelSslBumping)
-            || (serverBump() && pinning.serverConnection))
-#endif
-           ;
+    if (!Config.accessList.on_unsupported_protocol)
+        return false;
+ #if USE_OPENSSL
```

Comment:
missing empty line here.
```suggestion

 #if USE_OPENSSL
```

Reply:
addressed at 27122e